lib/screens/transfer/receive/receive_enter_data/interactor/receive_enter_data_bloc.dart

```diff
@@ -25,8 +24,7 @@ class ReceiveEnterDataBloc extends Bloc<ReceiveEnterDataEvents, ReceiveEnterData
     } else if (event is OnAmountChange) {
       final double parsedQuantity = double.tryParse(event.amountChanged) ?? 0;
 
-      final String seedsToFiat =
-          state.ratesState.fromSeedsToFiat(parsedQuantity, settingsStorage.selectedFiatCurrency).fiatFormatted;
+      final double seedsToFiat = state.ratesState.fromSeedsToFiat(parsedQuantity, settingsStorage.selectedFiatCurrency);
 
       if (parsedQuantity > 0) {
         yield state.copyWith(isNextButtonEnabled: true, quantity: parsedQuantity, fiatAmount: seedsToFiat);
```

Comment:
This should not pass around doubles either but maybe keep that for another PR (seedsToFiat being a double)